WORKSPACE

```diff
@@ -1,9 +1,8 @@
-new_http_archive(
+new_git_repository(
```

Comment:
@Mistobaan does this change actually work? We originally were using new_http_archive because we needed the strip_prefix option to remove the outer 'pretty_midi' directory.